src/shared/domain/search/search-result-view-item.model.ts

```diff
@@ -1,8 +1,11 @@
 import { CaseField } from '../definition';
+import { Type } from 'class-transformer';
 
+// @dynamic
```

Comment:
is this annotation needed?

Reply:
Yes

Reply:
Else you get lambda expression compile error when using class transformer